dora/stats/views.py

```diff
@@ -43,6 +43,7 @@ def get_categories(cats_values, subcats_values):
     service_slug = request.data.get("service", "")
     structure_slug = request.data.get("structure", "")
     service = structure = orientation = None
+    service_mobilisable = request.data.get("mobilisable", "")
```

Comment:
```suggestion
    service_mobilisable = request.data.get("mobilisable", "") in TRUTHY_VALUES
```
ça te donne directement un booleen que tu peux utiliser tel quel l. 107, et ça évite que qqun passe "0" et que ça soit interprété comme true...